source/_components/sensor.ebusd.markdown

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: "ebusd"
+description: "The ebusd sensor allows the integration between eBUS heating system and Home Assistant."
+date: 2018-10-01 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
+ha_release: "0.80"
+---
+
+Integration between ebusd (https://github.com/john30/ebusd), daemon for communication with eBUS heating systems, and homeassistant using sensor component.
```

Comment:
This is probably the line causing the error in netlify. Could you try to format the line as a markdown link? Should be something like [ebusd](https://github.com/john30/ebusd)